src/main/java/space/obminyashka/items_exchange/dto/UserChangeEmailDto.java

```diff
@@ -17,14 +16,5 @@ public class UserChangeEmailDto {
 
     @NotEmpty(message = ResponseMessagesHandler.ValidationMessage.INVALID_NOT_EMPTY)
     @Email(regexp = PatternHandler.EMAIL, message = ResponseMessagesHandler.ValidationMessage.INVALID_EMAIL)
-    private String newEmail;
-
-    @NotEmpty(message = ResponseMessagesHandler.ValidationMessage.INVALID_NOT_EMPTY)
-    private String newEmailConfirmation;
-
-    @SuppressWarnings("unused")                         // Used in validation process by Spring Validator
-    @AssertTrue(message = "{" + ResponseMessagesHandler.ValidationMessage.INVALID_CONFIRM_EMAIL + "}")
-    private boolean isEmailEquals() {
-        return newEmail.equals(newEmailConfirmation);
-    }
+    private String email;
```

Comment:
It looks like the DTO right now has only one field. Why don't you replace it with a request parameter instead and inform FE about such a change?

Reply:
Deleted DTO